### PR TITLE
Jon's User Guide section

### LINKED ISSUE #26: Update Jon's section for the User Guide
Adding and viewing Staff Schedules

### PR DESCRIPTION
**Pull request by fullfatwasabi**
closes #26

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -21,29 +21,27 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="images/janedoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
 
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Jonathan Tan
```
Comment (Gabau) [inline docs/AboutUs.md:32]: Is there a need to do About Us in this PR?

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM, but why are there two PRs addressing the AboutUs?

File: docs/AboutUs.md (hunk 1 above)
Comment (fullfatwasabi) [reply docs/AboutUs.md]: yep was an accident haha

**Review by Gabau — APPROVED** [review]
LGTM

**Comment by mweeruien** [comment]
LGTM!

**Pull request merged**